diff --git a/scripts/Makefile.lib b/scripts/Makefile.lib
index bdf42fd..8f93d8d 100644
--- a/scripts/Makefile.lib
+++ b/scripts/Makefile.lib
@@ -239,7 +239,8 @@ cmd_objcopy = $(OBJCOPY) $(OBJCOPYFLAGS) $(OBJCOPYFLAGS_$(@F)) $< $@
 # ---------------------------------------------------------------------------
 
 quiet_cmd_gzip = GZIP    $@
-cmd_gzip = (cat $(filter-out FORCE,$^) | gzip -n -f -9 > $@) || \
+cmd_gzip = (cat $(filter-out FORCE,$^) | gzip -n -f -9 > $@) && \
+	( which advdef > /dev/null && advdef -z4 $@ ; true ) || \
 	(rm -f $@ ; false)
 
 # DTC
